src/hoc/modals/ReadPillModal/ElementTypeRender/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import EllipseIcon from '../../../../assets/icons/EllipseIcon';
+import { SquaredIcon } from '../../../../assets/icons/SquaredIcon';
+import {
+  StyledColumn,
+  StyledImage,
+  StyledRow,
+  StyledText,
+} from '../../../../components/styled/styles';
+import { removeHtmlTags } from '../../../../utils/utils';
+
+export interface BubbleProps {
+  type: string;
+  id: string;
+  metadata?: {
+    metadata?: { lerni_question_type: string };
+    options?: string[];
+  };
+  name: string;
+  question_type?: string;
+}
+
+export const ElementTypeRender = ({ type, id, metadata, name, question_type }: BubbleProps) => {
+  if (type === 'ACTION') {
+    if (!metadata) {
+      return (
+        <StyledColumn css={{ justifyContent: 'center', alignItems: 'left', gap: '4px' }}>
+          <StyledText variant="body3" color="primary500" css={{ fontFamily: 'Roboto-Bold' }}>
+            {'Text'}
+          </StyledText>
+          <StyledText variant="body2" color="gray950">
+            {removeHtmlTags(name)}
+          </StyledText>
+        </StyledColumn>
+      );
+    } else if (metadata && metadata.metadata && metadata.metadata.lerni_question_type === 'image') {
+      return (
+        <StyledColumn css={{ justifyContent: 'center', alignItems: 'left', gap: '4px' }}>
+          <StyledText variant="body3" color="primary500" css={{ fontFamily: 'Roboto-Bold' }}>
+            {'Imagen'}
```

Comment:
I would change the rest of the labels to Spanish (like this one) to be consistent with the platform language.

Reply:
In fact, Figma shows labels in Spanish so we should change the others.